app/bundles/CoreBundle/Assets/css/app.css

```diff
@@ -1152,11 +1152,11 @@ h6,
   text-transform: uppercase;
 }
 .text-muted {
-  color: #8393a2;
+  color: #5d6c7c;
```

Comment:
This change needs to be done in `CoreBundle/Assets/css/app/less/components/type.less:101`.